packages/ui/src/components/MenuV2/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const SIZES = {
+  small: '180px',
+  medium: '280px',
+  large: '380px',
```

Comment:
just a question, not an ask for changes:  no REM sizes here ? I saw a min-width of 20rem

Reply:
yes you are correct